server/ocsp.go

```diff
@@ -162,15 +163,40 @@ func (oc *OCSPMonitor) getRemoteStatus() ([]byte, *ocsp.Response, error) {
 		overrideURLs = config.OverrideURLs
 	}
 	getRequestBytes := func(u string, hc *http.Client) ([]byte, error) {
+		start := time.Now()
 		resp, err := hc.Get(u)
 		if err != nil {
 			return nil, err
 		}
 		defer resp.Body.Close()
+
+		oc.srv.Debugf("Received OCSP response (method=GET, status=%v, url=%s, duration=%.3fs)",
+			resp.StatusCode, u, time.Since(start).Seconds())
 		if resp.StatusCode != http.StatusOK {
 			return nil, fmt.Errorf("non-ok http status: %d", resp.StatusCode)
 		}
+		return io.ReadAll(resp.Body)
+	}
+	postRequestBytes := func(u string, body []byte, hc *http.Client) ([]byte, error) {
+		hreq, err := http.NewRequest("POST", u, bytes.NewReader(body))
+		if err != nil {
+			return nil, err
+		}
+		hreq.Header.Add("Content-Type", "application/ocsp-request")
+		hreq.Header.Add("Accept", "application/ocsp-response")
+
+		start := time.Now()
+		resp, err := hc.Do(hreq)
+		if err != nil {
+			return nil, err
+		}
+		defer resp.Body.Close()
 
+		oc.srv.Debugf("Received OCSP response (method=POST, status=%v, url=%s, duration=%.3fs)",
+			resp.StatusCode, u, time.Since(start).Seconds())
+		if resp.StatusCode != http.StatusOK {
```

Comment:
Shouldn't this be `if resp.StatusCode > 299` to return an error and then perhaps if it's not 200 then log it as being of interest but not an error?

Reply:
I will make this change.